model/src/main/proto/topic.proto

```diff
@@ -247,8 +247,16 @@ enum ChapterPlayState {
   // The chapter has been started but not completed by the player.
   STARTED_NOT_COMPLETED = 3;
 
+  // The chapter has been started but not completed by the player and the progress made in the
+  // chapter is saved.
+  IN_PROGRESS_SAVED = 4;
+
+  // The chapter has been started but not completed by the player and the progress made in the
+  // chapter is not saved.
+  IN_PROGRESS_NOT_SAVED = 5;
+
   // The chapter has been completed by the player.
-  COMPLETED = 4;
+  COMPLETED = 6;
```

Comment:
Why are we not adding the new states after COMPLETED?  (i.e. IN_PROGRESS_SAVED = 5 and IN_PROGRESS_NOT_SAVED = 6)In general we should avoid making updates to existing proto fields

Reply:
moved the two new states after COMPLETED.